src/updaters/changelog-json.ts

```diff
@@ -65,13 +66,23 @@ export class ChangelogJson extends DefaultUpdater {
     logger.info(`adding release ${this.version} for ${this.artifactName}`);
     const changes = [];
     for (const commit of this.commits) {
+      const issues = commit.references.map(ref => ref.issue);
       // The commit.message field contains the type/scope prefix.
-      const message = commit.message.replace(COMMIT_PREFIX, '');
+      let message = commit.message.replace(COMMIT_PREFIX, '');
+      // When squashing commits, GitHub adds a suffix refrencing
+      // the # of the PR, e.g., chore(main): release 15.5.1 (#1838)
+      // this logic removes this suffix and prepends it to the
+      // issues array.
+      const match = message.match(PR_SUFFIX_REGEX);
+      if (match && match.groups?.pr) {
+        message = message.replace(match[0], '');
+        issues.unshift(match.groups.pr);
+      }
```

Comment:
Is it possible that we might need to de-dup the issue refs?

Reply:
Good question, probably worth doing so just in case?

Reply:
I've switched to using a `Set`, to eliminate duplicates.